.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Ruby",
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {},
+    "ghcr.io/devcontainers/features/go:1": {}
+  },
+  "customizations": {
+    "codespaces": {
+      "openFiles": ["CONTRIBUTING.md", "lib/linguist/languages.yml"]
```

Comment:
**Extremely trivial nitpick:** these arrays should be formatted one entry per line, both for consistency and ease-of-diffing.

```suggestion
      "openFiles": [
        "CONTRIBUTING.md",
        "lib/linguist/languages.yml"
      ]
```